dbms/src/Storages/MergeTree/IMergedBlockOutputStream.h

```diff
@@ -141,7 +141,10 @@ class IMergedBlockOutputStream : public IBlockOutputStream
     size_t aio_threshold;
 
     size_t current_mark = 0;
-    size_t skip_index_mark = 0;
+
+    /// Number of mark in data from which skip indices have to start
```

Comment:
I wonder why Yandex C++ style guide requires triple slashes for line comments, interesting

Reply:
It's not Yandex style guide, just [ClickHouse](https://github.com/yandex/ClickHouse/blob/master/docs/ru/development/style.md#%D0%BA%D0%BE%D0%BC%D0%BC%D0%B5%D0%BD%D1%82%D0%B0%D1%80%D0%B8%D0%B8)  :)

As far as I know, It's default comments style for [KDevelop IDE](https://www.kdevelop.org/) which is widely used for ClickHouse development.

Reply:
Thank you for explaining, much appreciated!